cmake/Dependencies.cmake

```
@@ -1560,7 +1560,7 @@ if(USE_KINETO)
     message(STATUS "Using Kineto with Roctracer support")
   endif()
 
-  if(NOT USE_XPU)
+  if((NOT USE_XPU) OR MSVC)
```

One in theory can use clang to compile for windows, than `MSVC` would be false, but `WIN32` would be true regardless of what compiler is used 
```suggestion
  if((NOT USE_XPU) OR WIN32)
```

---

Makes sense, thanks.